src/WinDebloat/WinGet.cs

```diff
@@ -63,14 +63,25 @@ public static async Task<List<Package>> List()
             if (line.Length > 35)
             {
                 list.Add(new(
-                    line[..35].TrimEnd(),
```

Comment:
why was this required?

Reply:
Without it, the id value for in the list had leading spaces, i.e. `          github.cli`

Reply:
ahh ok. thanks